packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js

```diff
@@ -201,7 +201,7 @@ MeetingInfoUtil.getRequestBody = (options) => {
       break;
     case _PERSONAL_ROOM_:
       body.userId = destination;
-      body.orgId = ''; // TODO: when to use the org iD (fetch others PMR)
+      body.orgId = webex.credentials.getOrgId(); // TODO: when to use the org iD (fetch others PMR)
```

Comment:
this TODO comment needs updating

Reply:
@marcin-bazyl can you elaborate on what needs to be updated here?

Reply:
This is not my todo comment, it was introduced by @arun3528 in https://github.com/webex/webex-js-sdk/pull/2174/ so it's best for Arun to comment, but I think we need to check if this fix that we're doing here works for fetching PMR of people outside of your org. If it does, then the todo comment can be removed. If it doesn't then a jira should be raised to fix that.

Reply:
I confirmed with the API team that the org Id that we need to send should correspond to that of the user for which we are trying to fetch PMR information. This code won't work for external users

Reply:
@taymoork2  can you remove this code we can add that in the next one